File: PSET Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int height;

int main(void)
{
    do
    {
        //get input from user
        height = get_int("Height: ");
    }
    //continue asking for inputs until you get a positive number between 1 and 8
    while (height <= 0 || height > 8);

    //to align blocks to the right
    for (int i = 0; i < height; i++)
    {
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        //to print blocks
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = i + 1; l > i; l--)
        {
            for (int m = 0; m < i + 1; m++)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: PSET Week 6/sentimental-mario-more/mario.py
def main():
    # Use height to build pyramid
    height = get_height()
    for i in range(height):
        for j in range(height - i - 1, 0, -1):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")
        print("  ", end="")
        for l in range(i + 1):
            print("#", end="")
        print()


# No use of cs50 library, ask user for input
def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                break
        # If not a number, continue asking
        except ValueError:
            n = int(input("Height: "))
    return n


main()